data/blog/posts/2022/2022-07-04-how-to-automatically-add-return-type-declarations-without-breaking-your-code.md

```diff
@@ -165,28 +165,22 @@ Depending on what age your project is coming from, there is one requirement to m
 
 ## Try it Yourself
 
-For each case above, there is one rector rule that handles this case. Add these 4 rules to your code and see for yourself:
+Rector can handles few cases above in its public version. Add these rules to your code and see for yourself:
 
 ```php
 use Rector\Config\RectorConfig;
 use Rector\CodeQuality\Rector\ClassMethod\ReturnTypeFromStrictScalarReturnExprRector;
-use Rector\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictBoolReturnExprRector;
 use Rector\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictNativeFuncCallRector;
-use Rector\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictNewArrayRector;
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->rules([
-        ReturnTypeFromStrictBoolReturnExprRector::class,
         ReturnTypeFromStrictNativeFuncCallRector::class,
```

Comment:
I recheck, it seems `ReturnTypeFromStrictNativeFuncCallRector` no longer exists in any repo, even rector-src

Reply:
I checked and it was renamed to `ReturnTypeFromStrictNativeCallRector` to handle all the calls.
Part of `rector-src`. I'll update the name here